tempesta_fw/classifier/frang.c

```diff
@@ -541,18 +551,21 @@ frang_http_host_check(const TfwHttpReq *req, FrangAcc *ra)
 	return ret;
 }
 
+/**
+ * Monotonically increasing time quantums. The configured time frame is devided
+ * by FRANG_FREQ slots to get the quantums granularity.
+ */
 static unsigned int
 frang_resp_quantum(void)
 {
-	return ((jiffies / HZ) % FRANG_RESP_TIME_PERIOD) * FRANG_FREQ
-		/ frang_cfg.http_resp_code_block->tf;
+	return jiffies / HZ * FRANG_FREQ / frang_cfg.http_resp_code_block->tf;
```

Comment:
In current implementation if `frang_cfg.http_resp_code_block->tf == 1`, then `ts % FRANG_FREQ` will always give 0.
It seems that `jiffies * FRANG_FREQ / (frang_cfg.http_resp_code_block->tf * HZ)` would produce better granularity.